config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/BundleFileBuilder.java

```diff
@@ -83,8 +91,13 @@ void buildBundle(File rootDir, File outputDir, List<File> dependencies, String n
         }
 
         //Zip
-        Element bundleElement = bundleEntityBuilder.build(bundle, EntityBuilder.BundleType.DEPLOYMENT, document);
-        documentFileUtils.createFile(bundleElement, new File(outputDir, name + ".bundle").toPath());
+        final Map<String, Pair<Element, BundleMetadata>> bundleElementMap = bundleEntityBuilder.build(bundle,
+                EntityBuilder.BundleType.DEPLOYMENT, document, projectName, projectGroupName, projectVersion);
+        for (Map.Entry<String, Pair<Element, BundleMetadata>> entry : bundleElementMap.entrySet()) {
+            documentFileUtils.createFile(entry.getValue().getLeft(),
+                    new File(outputDir, entry.getKey() + ".bundle").toPath());
+            jsonFileUtils.createBundleMetadataFile(entry.getValue().getRight(), entry.getKey(), outputDir);
```

Comment:
Can you post some examples of the file names along with the extensions?
I suppose you are following the namings as per the document: https://docs.google.com/document/d/17x1AnwYQc2mF6AENcCGYfNN0SjLtOyh4lZ8nj0VIgLA/edit#heading=h.3wekv5psqijk

Reply:
Yes, I am following naming convention mentioned in the doc. Where should I provide the examples? In code comment or the doc?